packages/mdc-bottom-line/_mixins.scss

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Public
+
+@mixin mdc-bottom-line-color($color) {
+  .mdc-bottom-line {
+    @include mdc-theme-prop(background-color, $color);
+  }
+}
+
+// Private
+
+@mixin mdc-bottom-line-trigger-element_($element) {
```

Comment:
maybe just `mdc-bottom-line($focusable-element)`. 

And make it a public mixin. Basically any other component which uses mdc-bottom-line, should specify the focusable element that goes with it. You should add a line to the README about *always* calling `mdc-bottom-line($focusable-element)`.